examples/component-program/consumer/Pulumi.yaml

```diff
@@ -0,0 +1,22 @@
+name: consume-go-component
```

Comment:
might be nice to set this up to be exercised by the example tests too.

Reply:
They already are 😄

Reply:
Ah you're right I just misread the make file